src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -86,7 +86,11 @@ export const JazzBar: React.FC<JazzProps> = ({ venue }) => {
   }, [analytics, seatedAtTable]);
 
   const shouldShowReactions =
-    seatedAtTable && areSettingsLoaded && settings.showReactions;
+    seatedAtTable &&
+    areSettingsLoaded &&
+    settings.showReactions &&
+    venue.showReactions;
+
```

Comment:
This looks like a better place to provide defaults in case of null/undefined

Reply:
yep, done 17a8f28